documentation/modules/about-cold-warm-migration.adoc

```diff
@@ -47,3 +47,83 @@ The precopy stage runs until the cutover stage is started manually or is schedul
 The VMs are shut down during the cutover stage and the remaining data is migrated. Data stored in RAM is not migrated.
 
 You can start the cutover stage manually by using the {project-short} console or you can schedule a cutover time in the `Migration` manifest.
+
+
+[id="warm-migration-versus-cold-migration"_{context}]
+== Advantages and disadvantages of cold and warm migrations
+
+[id="warm-cold-migration-overview_{context}"]
+=== Overview
+
+Both cold migration and warm migration have advantages and disadvantages, as described in the table that follows:
+
+[cols="1,1,1",options="header"]
+.Advantages and disadvantages of cold and warm migrations
+
+|===
+| |Duration |VM downtime
+
+|Cold migration
+|Minimal
+|High
+
+|Warm migration
+|Longer and variable
+|Minimal
+|===
+
+[id="warm-cold-migration-detailed_{context}"]
+=== Detailed description
+
+The table that follows offers a more detailed description of the advantages and disadvantages of each type of migration. It assumes that you have installed Red Hat Enterprise Linux (RHEL) 9 on the {ocp} platform on which you installed {project-short}.
+
+[cols="1,1,1",options="header"]
+.Detailed description of advantages and disadvantages
+
+|===
+| |Cold migration |Warm migration
+
+|*Duration*
```

Comment:
Cold
Duration: Driven by amount of data on disks
Data transferred: Approx sum of all disks
…

Warm
Duration: Driven by amount of data on disks and VM utilization
Data transferred: Approx sum of all disks and VM utilization
…

Reply:
OK